File: C-Python_Solving_50_Problems/16-Rectangle_Area_Using_Diagonal.c
#include <stdio.h>
#include <math.h>

int main() {
    float a, d, area;

    // take inputs
    printf("Enter the length of one side of the rectangle: ");
    scanf("%f", &a);
    printf("Enter the length of the diagonal of the rectangle: ");
    scanf("%f", &d);

    // calculate area
    area = a * sqrt(d*d - a*a);

    // display result
    printf("The area of the rectangle is %.3f", area);

    return 0;
}

File: C-Python_Solving_50_Problems/16-Rectangle_Area_Using_Diagonal.py
# Python program to calculate the area of a rectangle using diagonal and one side
# take inputs
a = float(input('Enter the length of one side of the rectangle: '))
d = float(input('Enter the length of the diagonal of the rectangle: '))

# calculate area
area = a * ((d**2 - a**2)**0.5)

# display result
print('The area of the rectangle is {0:.3f}'.format(area))
